sql/parse/parse_test.go

```diff
@@ -2873,6 +2873,20 @@ CREATE TABLE t2
 				"foo",
 			),
 		},
+		{
+			input: `alter table t add index (i), drop index i, add check (i = 0), drop check chk, drop constraint c, add column i int, modify column i text, drop column i, rename column i to j`,
```

Comment:
if i did something to break this test, is there a docstring you could add to `convertMultiAlterDDL` that would help me figure out the precedence order? It seems like the order of top-level case statements are the precedence order? and there's no real mysql docs for this behavior?